rllib/core/models/catalog.py

```diff
@@ -224,3 +240,80 @@ def get_tokenizer_config(
                 **{"use_lstm": False, "use_attention": False},
             },
         )
+
+    @classmethod
+    def get_action_dist_cls_dict(
```

Comment:
Should this just take framework as input? and return the corresponding the  dist class? instead of a mapping from fw to the class?